### ISSUE TITLE
How can I render children inside the react component?

### ISSUE THREAD

**Issue by aplaskota** [coatue-oss/react2angular#136]
Hi, I'm trying to figure out how can I make my react components more flexible. I'd like to achieve something like this inside my angular.js template.

```
<div>
    <myReactComponent>
        <p>Text<p/>
    </myReactComponent>
</div>
```

**Comment by EmBeaux** [comment]
Is this something you were able to figure out? I am running in to the same problem of trying to utilize `ng-content` within react.

**Comment by aplaskota** [comment]
@EmBeaux nope, this is the biggest constraint in this package so I'm considering other options right now.
